presto-main/src/main/java/io/prestosql/execution/SqlQueryExecution.java

```diff
@@ -358,10 +358,10 @@ public void addFinalQueryInfoListener(StateChangeListener<QueryInfo> stateChange
         stateMachine.addQueryInfoStateChangeListener(stateChangeListener);
     }
 
-    private PlanRoot analyzeQuery()
+    private PlanRoot logicalPlan()
```

Comment:
I'm not sure `logicalPlan` is the right name here?  I'm not super familiar with the terms used for planning, but method captures the initial plan building and all of the optimizations.  The only thing sit doesn't capture is analysis and distributed planning (i.e., fragmentation).  @martint or @findepi thoughts?

Reply:
It should just be called “planQuery”. The stat should be called “planningTime”

Reply:
The problem here is that there is also `PLANNING` query state so planning time is currently measured and it tracks time spent in this state. Notice that during that state we do `analyzeQuery/logicalPlan` and `planDistribution`.

Reply:
Maybe we could remove this measurement at all, because it is already captured by time spent in `PLANNING` state? 

Then what about measuring time for `planDistribution`. In the first place I don't know why it was captured. Where there any issues with this?

Reply:
> Then what about measuring time for planDistribution. In the first place I don't know why it was captured. Where there any issues with this?

Originally. it included some interactions with HDFS for enumerating splits, so that's why we measured it. I don't think it matters anymore, so we could remove it.